Examples/ScriptLibrary/LibraryBase.cs

```diff
@@ -226,6 +226,8 @@ public void UnsubscribeToSimpleEvent(int eventCount)
 
         public sealed override void Init()
         {
+            PreSimpleInit();
```

Comment:
Move this down below starting the FindDebugger coroutine, before the Yield(); That will get the name setup and start working on finding the debugger.